### PR TITLE
feat(cypress): Test locations & roles display

### LINKED ISSUE #697: Support viewing metadata locations
### Description

As a catalog author, I want to view locations in regard to metadata.

### Acceptance Criteria

* The `location`s field and it's respective metadata is properly displayed within the React library. A `location`s metadata include:
   * `uuid` (not displayed to author)
   * `title` [1]
   * `address` [1]
   * `email-address` [0 to many)
   * `telephone-number` [0 to many)
   * `url` [0 to many)
   * `prop` [0 to many)
   * `link` [0 to many)
   * `remarks` [0 to many)

### Proposed Solution

_No response_

### Additional Notes

A lot of this should already be support. Some of a location's metadata like `link` and `remarks` are not currently supported.

### Implementation

- [ ] I can (or plan to) submit a pull request to implement this
- [ ] Implementing this may result in a breaking change

### PR DESCRIPTION
**Pull request by tuckerzp**
To avoid regression after https://github.com/EasyDynamics/oscal-react-library/pull/728
is merged in, this tests that locations and roles display properly.

It will require changes from https://github.com/EasyDynamics/oscal-demo-content/pull/35
for the tests to pass.

### CODE REVIEW

File: end-to-end-tests/cypress/support/commands.js [hunk 1]
```diff
@@ -112,3 +112,10 @@ Cypress.Commands.add("setTestSspJson", (sspJson) => {
   );
 });
 
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/
+Cypress.on('uncaught:exception', (err) => {
+    /* returning false here prevents Cypress from failing the test */
+    if (resizeObserverLoopErrRe.test(err.message)) {
+        return false
+    }
+})
```
Comment (kylelaker) [inline end-to-end-tests/cypress/support/commands.js:121]: Why isn't this being resolved with the code that already exists?

https://github.com/EasyDynamics/oscal-editor-deployment/blob/a3bc3798b87220fec211554efdb443a558443874/end-to-end-tests/cypress/support/e2e.js#L22-L36

Comment (tuckerzp) [reply end-to-end-tests/cypress/support/commands.js:121]: Excellent question. I will look into that.

File: end-to-end-tests/cypress/support/e2e.js [hunk 2]
```diff
@@ -19,18 +19,20 @@ import './commands'
...
   }
+
   if (err.message.includes("Cannot read properties of null (reading 'getText')")) {
     return false
   }
+  
   // TODO: Find a fix where we don't need to avoid this exception
   // https://github.com/EasyDynamics/oscal-editor-deployment/issues/121
-  if (err.message.includes("ResizeObserver loop limit exceeded")) {
+  if (err.message.includes("ResizeObserver loop")) {
```
Comment (tuckerzp) [inline end-to-end-tests/cypress/support/e2e.js:35]: Looking at the cypress output, the error message did not always match "ResizeObserver loop limit exceeded". It always has "ResizeObserver loop", so I just shorted the `includes`.

**Review by laurelmay — APPROVED** [review]
Great work on these tests, @tuckerzp!

**Pull request merged**